STM32Cube/Tasks/health_checks.c

```diff
@@ -84,6 +84,16 @@ void healthCheckTask(void *argument)
     current_data[1] = adc1_current_mA & 0xFF;
     build_board_stat_msg(millis_(), E_5V_OVER_CURRENT, current_data, 2, &msg);
     xQueueSend(busQueue, &msg, 10);
+    } else {
+    // E nominal
+    can_msg_t msg;
+    build_board_stat_msg(0, E_NOMINAL, NULL, 0, &msg);
+    xQueueSend(busQueue, &msg, 10);
+
+    // Current Draw Message
+    build_analog_data_msg(0, SENSOR_BATT_CURR, adc1_current_mA, &msg);
```

Comment:
add a call to millis_() instead of 0
ad1_current_mA needs to be formatted into a uint8_t. See the overcurrent error message for how that should be done